### ISSUE TITLE
[Chapter-10] Component ID generation collision 

### ISSUE THREAD

**Issue by alx-andru** [PacktPublishing/Next-Level-UI-Development-with-PrimeNG#8]
When running the example of Chapter 10 running into following warning in the console: 

<img width="649" alt="Screenshot 2024-01-21 at 5 36 32 PM" src="https://github.com/PacktPublishing/Next-Level-UI-Development-with-PrimeNG/assets/19511694/c96a0af6-8258-4013-b24f-0823ae687531">

> NG0912: Component ID generation collision detected. Components 'HomeComponent' and 'UserProfileComponent' with selector 'ng-component' generated the same component ID. To fix this, you can change the selector of one of those components or add an extra host attribute to force a different ID. Find more at https://angular.io/errors/NG0912

**Comment by dalenguyen** [comment]
Yeah, that was introduced recently https://github.com/angular/angular/issues/50158.
